### DISCUSSION TITLE
How to lock out aliases you don't want to hear

### DISCUSSION THREAD

**Discussion by Bonk01** [DSheirer/sdrtrunk#1888] [Q&A]
I slid the listen slider to off on aliases that I didn't want to hear but they still come through even when you put in a site with or without aliases

**Comment by DSheirer** [comment]
Can you please clarify what you mean by "...even when you put in a site with or without aliases"?

Make sure the alias list is attached to the channel in the playlist channel configuration editor.  Make sure the aliases are working.  Are the talkgroup and radio values aliasing correctly?  Are the alias labels showing up next to the values in the now playing and events views?

**Reply by Bonk01** [reply]
I put in a site to see what it would do I was getting talkgroups without putting talkgroups for that county, I have about 20 talkgroups I like to hear will need 20 alises do I import all talkgroups

**Comment by DSheirer** [comment]
(edited) The default behavior of sdrtrunk is to play all talkgroups and you don't need an alias list for that to happen.  You can use a talkgroup range to alias all talkgroup values 1-65535 and set them to default 'Do Not Monitor'.  Then, you can create individual talkgroup aliases for the talkgroups that you do want to listen to.  Indiivdual talkgroup take precedence over talkgroup ranges, so the talkgroups that you want to listen to will alias match correctly against the individual talkgroup aliases that you create and everything else will default match to the talkgroup range.

**Reply by Bonk01** [reply]
Put them by hand
